builtin/logical/pki/path_tidy.go

```diff
@@ -62,42 +64,51 @@ type tidyStatus struct {
-	IssuerAssocs       bool          `json:"tidy_revoked_cert_issuer_associations"`
-	ExpiredIssuers     bool          `json:"tidy_expired_issuers"`
-	BackupBundle       bool          `json:"tidy_move_legacy_ca_bundle"`
-	SafetyBuffer       time.Duration `json:"safety_buffer"`
-	IssuerSafetyBuffer time.Duration `json:"issuer_safety_buffer"`
-	PauseDuration      time.Duration `json:"pause_duration"`
-	MaintainCount      bool          `json:"maintain_stored_certificate_counts"`
-	PublishMetrics     bool          `json:"publish_stored_certificate_count_metrics"`
-	RevocationQueue    bool          `json:"tidy_revocation_queue"`
-	QueueSafetyBuffer  time.Duration `json:"revocation_queue_safety_buffer"`
-	CrossRevokedCerts  bool          `json:"tidy_cross_cluster_revoked_certs"`
+	Enabled                 bool          `json:"enabled"`
+	Interval                time.Duration `json:"interval_duration"`
+	CertStore               bool          `json:"tidy_cert_store"`
+	RevokedCerts            bool          `json:"tidy_revoked_certs"`
+	IssuerAssocs            bool          `json:"tidy_revoked_cert_issuer_associations"`
+	ExpiredIssuers          bool          `json:"tidy_expired_issuers"`
+	BackupBundle            bool          `json:"tidy_move_legacy_ca_bundle"`
+	TidyAcme                bool          `json:"tidy_acme"`
+	SafetyBuffer            time.Duration `json:"safety_buffer"`
+	IssuerSafetyBuffer      time.Duration `json:"issuer_safety_buffer"`
+	AcmeAccountSafetyBuffer time.Duration `json:"acme_account_safety_buffer"`
+	PauseDuration           time.Duration `json:"pause_duration"`
+	MaintainCount           bool          `json:"maintain_stored_certificate_counts"`
+	PublishMetrics          bool          `json:"publish_stored_certificate_count_metrics"`
+	RevocationQueue         bool          `json:"tidy_revocation_queue"`
+	QueueSafetyBuffer       time.Duration `json:"revocation_queue_safety_buffer"`
+	CrossRevokedCerts       bool          `json:"tidy_cross_cluster_revoked_certs"`
 }
 
 var defaultTidyConfig = tidyConfig{
-	Enabled:            false,
-	Interval:           12 * time.Hour,
-	CertStore:          false,
-	RevokedCerts:       false,
-	IssuerAssocs:       false,
-	ExpiredIssuers:     false,
-	BackupBundle:       false,
-	SafetyBuffer:       72 * time.Hour,
-	IssuerSafetyBuffer: 365 * 24 * time.Hour,
-	PauseDuration:      0 * time.Second,
-	MaintainCount:      false,
-	PublishMetrics:     false,
-	RevocationQueue:    false,
-	QueueSafetyBuffer:  48 * time.Hour,
-	CrossRevokedCerts:  false,
+	Enabled:                 false,
+	Interval:                12 * time.Hour,
+	CertStore:               false,
+	RevokedCerts:            false,
+	IssuerAssocs:            false,
+	ExpiredIssuers:          false,
+	BackupBundle:            false,
+	TidyAcme:                false,
+	SafetyBuffer:            72 * time.Hour,
+	IssuerSafetyBuffer:      365 * 24 * time.Hour,
+	AcmeAccountSafetyBuffer: 48 * time.Hour,
```

Comment:
Given that certbot doesn't gracefully handle account removal, I'd maybe make this 30 days perhaps by default.

Reply:
Gotcha.  I've done that.  Did you want this to apply to time after order/cert expiry as well, if certbot doesn't handle that well?